chainstate/src/detail/tests/processing_tests.rs

```diff
@@ -28,6 +28,8 @@ use common::chain::OutputSpentState;
 use common::chain::UpgradeVersion;
 use common::primitives::Compact;
 use common::Uint256;
+use crypto::key::KeyKind;
+use crypto::key::PrivateKey;
```

Comment:
Not a big deal, but both can be in one `use`

Reply:
Ok, I wouldn't mind combining them in one `use`.